delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/search/SearchResults.scala

```diff
@@ -28,16 +28,6 @@ sealed trait SearchResults[A] extends Product with Serializable {
   def token: Option[String]
   def results: Seq[ResultEntry[A]]
   def sources: Seq[A] = results.map(_.source)
-
-  /**
-    * Constructs a new [[SearchResults]] with the provided ''results''
-    *
-    * @param res
-    *   the provided collection of results
-    * @tparam B
-    *   the generic type of the newly created [[SearchResults]]
-    */
-  def copyWith[B](res: Seq[ResultEntry[B]]): SearchResults[B]
```

Comment:
Moved it to the test package because it wasn't used anywhere else. Can be reverted if needed